stork/models.py

```diff
@@ -315,6 +365,13 @@ def get_metrics_string(self, metrics_array, prefix="", postfix=""):
             s = s + " %s=%.3g" % (name, val)
         return s
 
+    def get_metrics_dict(self, metrics_array, prefix="", postfix=""):
```

Comment:
Looks good, but the metric dict functions could maybe be put into one at some point

Reply:
agreed. TBH I am not sure why we need so many functions for the metrics. 
For a future re-factor, we should think hard about whether the metrics should be tracked by the LossStack class at all or whether we can should make a metric tracker